clients/web/src/stylesheets/layout/global.styl

```diff
@@ -1,18 +1,16 @@
 /**
  * Place global styles in this file.
  */
-@import '~nib/index.styl'
-
-*:not(i)
```

Comment:
FYI, this is one substantive change. Rather than apply `font-family` to `*:not(i)`, we apply it to `body`, and allow it to propagate down. This has the same effect on the document, but is much more efficient.

Interestingly, the `:not(i)` never had any real effect, since the style applied to the parent of any `i` element would simply propagate down to the `i`.